### ISSUE TITLE
Notify user that changes are not persisted until form is sent

### ISSUE THREAD

**Issue by amiedes** [PopulateTools/gobierto#923]
For nested forms that make use of modals to edit a subset of the information, we should make clear to the user that changes made inside the modal won't be saved until the whole form is sent (click "Update").

The "save" button on the modal only persists the changes on the DOM, not in the DB.

![captura de pantalla 2017-09-06 a las 17 16 13](https://user-images.githubusercontent.com/9287468/30119795-267075aa-9327-11e7-8da6-0d40eab9ea32.png)

Affected screens:

* Create / edit of processes
* Create / edit of polls

Tasks:

* [ ] Design a flash message to alert the user (@furilo)
* [ ] Implement (@amiedes)

**Comment by furilo** [comment]
@Crashillo can you do a proposal as how you would do this? My idea was to show a flash like message saying something like: "These changes will be saved only when you Update the process"

**Comment by Crashillo** [comment]
Definitely, I'd use an alert banner. Perhaps, with a different color, even more highlighted. That banner would come up when modal is saved. Below is the easiest implementation:
![imagen](https://user-images.githubusercontent.com/817526/30913397-589ab616-a390-11e7-98d6-c0de31ac7875.png)

Due to the form height, it would be better if that banner was in the update sticky panel, as:
![imagen](https://user-images.githubusercontent.com/817526/30913857-e779f8d2-a391-11e7-8db9-842370f72bfb.png)

Another option is to place the banner as shown in the first image, but make it sticky all over the page. 

As well, we could inform in the modified form field itself:
![imagen](https://user-images.githubusercontent.com/817526/30914148-dfa16a36-a392-11e7-9b54-313552c5412d.png)
Not exclusively here, but inside the sticky also, to inform user where is/are the modifications.

Tell me your thoughts.

**Comment by amiedes** [comment]
* I like the sticky idea, since when a few questions have been added the banner won't be visible unless we do this.
* In case we make it sticky, I'd put it in the update panel, otherwise maybe it's too "*obtrusive*"
* I also think this will be really useful:

![image](https://user-images.githubusercontent.com/9287468/30914707-dbe692d4-a394-11e7-9b87-8d1ac3e617de.png)

**Comment by furilo** [comment]
Let's go with the flash inside the sticky panel. 

But avoid the icon for the time being (until we start using them in other places as well), and maybe make the font size the same as in Borrador / Publico ?
